test/integration/scheduler/plugins/plugins_test.go

```diff
@@ -1670,6 +1689,7 @@ func TestBindPlugin(t *testing.T) {
 			defer teardown()
 
 			pluginInvokeEventChan = make(chan pluginInvokeEvent, 10)
+			defer close(pluginInvokeEventChan)
```

Comment:
need to close the channel to let all the plugins know the test is complete.
otherwise we'd hit `found unexpected goroutines` error, which tell us there's some in-progress goroutine.